### PR TITLE
Fix broken badges and link on readme page

### LINKED ISSUE #249: Broken badges on README page
image

### PR DESCRIPTION
**Pull request by northwestwitch**
### This PR adds | fixes:
- Remove the broken conda badge on readme page
- Fixed the link of the other broken badges on readme (fix #249)
- Fixed a broken link on readme

### How to test:
- Check how the page looks with a markdown preview

### Expected outcome:
- [x] The badges should be working

### Review:
- [x] Code approved by DNIL
- [x] Tests executed by automatic tests

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by northwestwitch** [comment]
Fix preview:

image

File: README.md [hunk 1]
```diff
@@ -1,10 +1,14 @@
 <p align="center">
-  <a href="http://chanjo.co">
```
Comment (dnil) [inline README.md:2]: 👀

File: README.md [hunk 2]
```diff
@@ -1,10 +1,14 @@
...
+  <a href="https://github.com/Clinical-Genomics/chanjo">
     <img height="235" width="244" src="docs/assets/logo.png"/>
   </a>
 </p>
 
-# Chanjo [![PyPI version][fury-image]][fury-url] [![Build Status][travis-image]][travis-url] [![Coverage Status][coveralls-image]][coveralls-url] [![bioconda-badge][bioconda-img]][bioconda-url]
+# Chanjo 
+![Docker build - GitHub](https://github.com/Clinical-Genomics/chanjo/actions/workflows/docker_build_n_publish.yml/badge.svg)
+[![PyPI version](https://badge.fury.io/py/chanjo.svg)](https://badge.fury.io/py/chanjo)
```
Comment (dnil) [inline README.md:9]: 👍

File: README.md [hunk 3]
```diff
@@ -1,10 +1,14 @@
 <p align="center">
-  <a href="http://chanjo.co">
+  <a href="https://github.com/Clinical-Genomics/chanjo">
     <img height="235" width="244" src="docs/assets/logo.png"/>
   </a>
 </p>
 
-# Chanjo [![PyPI version][fury-image]][fury-url] [![Build Status][travis-image]][travis-url] [![Coverage Status][coveralls-image]][coveralls-url] [![bioconda-badge][bioconda-img]][bioconda-url]
+# Chanjo 
+![Docker build - GitHub](https://github.com/Clinical-Genomics/chanjo/actions/workflows/docker_build_n_publish.yml/badge.svg)
```
Comment (dnil) [inline README.md:8]: I don't see an image for this, but maybe it has just been purged since no updates, and will manifest on next run?

**Review by dnil — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (northwestwitch) [reply README.md:8]: That's because the action never had a run: https://github.com/Clinical-Genomics/chanjo/actions/workflows/docker_build_n_publish.yml

**Pull request merged**